makefile.shade

```diff
@@ -5,16 +5,6 @@ k-standard-goals
 
 default DOTNET_ARGUMENT='-v '
 
-#build-compile target='compile'
```

Comment:
Who uses the stuff under `experimental`? One of the projects (`BasicAsyncHttpServer`) is actually empty.